src/nvector/pthreads/fmod/fnvector_pthreads_mod.f90

```diff
@@ -762,20 +762,20 @@ function FN_VGetArrayPointer_Pthreads(v) &
 use, intrinsic :: ISO_C_BINDING
 real(C_DOUBLE), dimension(:), pointer :: swig_result
 type(N_Vector), target, intent(inout) :: v
-type(C_PTR) :: fresult 
-type(C_PTR) :: farg1 
+type(C_PTR) :: fresult
+type(C_PTR) :: farg1
 
 farg1 = c_loc(v)
 fresult = swigc_FN_VGetArrayPointer_Pthreads(farg1)
-call c_f_pointer(fresult, swig_result, [1])
+call c_f_pointer(fresult, swig_result, [FN_VGetLength_Pthreads(v)])
```

Comment:
`N_VGetLocalLength_MPIPlusX` is missing from the `.h` file

Reply:
Comment on the wrong file?

Reply:
Misread the function name, this is good.